frontend/src/containers/App.js

```diff
@@ -0,0 +1,194 @@
+import React, { useEffect, useRef } from 'react';
+
+import * as rmsClient from '../api/RoomzMediaServerClient.js'
+import './App.css';
+
+
+
+function App(props) {
+  const room_id = useRef();
+  const user_id = useRef();
+  const egress_media = useRef();
+
+  var ICE_SERVERS = [
+    {urls:"stun:stun.l.google.com:19302"}
+  ];
+  var roomyPcs = {};
+  let videos = [
+    { peer_id: "0-0" },
```

Comment:
I'm wondering if you should initialize this to something that is invalid, like `null`. Also not sure if this needs to be camelCare or not depending on how this interacts with the backend.